regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * preamble-head-view.js
+ *
+ * This view specifically handles Preamble actions in the
+ * Sub View Header, notably click actions of the read/write tabs.
+ *
+ */
+
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var CommentEvents = require('../../events/comment-events');
+
+var PreambleHeadView = Backbone.View.extend({
+  el: '.preamble-header',
```

Comment:
Is this selector unique? If it is, should we use an ID instead of a class to clarify that there should only be one of them?